# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1373749481 14400
#      Sat Jul 13 17:04:41 2013 -0400
# Node ID 47cc8e3d9183b6070f2cbfab194806cc44102537
# Parent  221e71d2aef0a3b69ff930bd042d3be7d283c703
Add demo to subplot.

script/plot/subplot.m: Add demo to illustrate bug # 39433.

diff --git a/scripts/plot/subplot.m b/scripts/plot/subplot.m
--- a/scripts/plot/subplot.m
+++ b/scripts/plot/subplot.m
@@ -359,8 +359,28 @@ endfunction
 %!   title (sprintf ('title (2,2,%d)', n));
 %! end
 %! subplot (1,2,1, 'align');
 %! plot (x, x);
 %! xlabel ('xlabel (1,2,1)');
 %! ylabel ('ylabel (1,2,1)');
 %! title ('title (1,2,1)');
 
+%!demo
+%! x = 0:10;
+%! subplot (221);
+%! plot (x, rand (3, 11))
+%! title ('x & y labels & ticklabels');
+%! xlabel xlabel
+%! ylabel ylabel
+%! subplot (222);
+%! plot (x, rand (3, 11))
+%! title ('no labels');
+%! axis ('nolabel','tic');
+%! subplot (223);
+%! plot (x, rand (3, 11))
+%! title ('no labels');
+%! axis ('nolabel','tic');
+%! subplot (224);
+%! plot (x, rand (3, 11))
+%! title ('x & y labels & ticklabels');
+%! xlabel xlabel
+%! ylabel ylabel
